Mono.Debugging/Mono.Debugging.Client/BreakpointStore.cs

```diff
@@ -483,10 +483,10 @@ static string ResolveFullPath (string path)
 			if (string.IsNullOrEmpty(path))
 				return path;
 
-			string result = null;
+			string result;
 			try
 			{
-				result = realpath(path, IntPtr.Zero);
+				result = realpath(path, IntPtr.Zero) ?? Path.GetFullPath(path);
```

Comment:
Nit: Could just do the Path.GetFullPath on the return statement. Less work for the JIT, and more likely to inline everything.

Reply:
```csharp
string result = null;
try { result = realpath(path, IntPtr.Zero); catch {}

return string.IsNullOrEmpty(result) ? Path.GetFullPath(path) : result;
```

Reply:
fixed